front/src/modules/simulationResult/components/SpaceTimeChart/drawTrain.ts

```diff
@@ -62,33 +48,17 @@ export default function drawTrain(
     d3.select(releventLine).attr('transform', `translate(${translation})`);
   };
 
-  let debounceTimeoutId: number;
-
-  function debounceUpdateDepartureArrivalTimes(
-    computedDepartureArrivalTimes: TrainsWithArrivalAndDepartureTimes[],
-    interval: number
-  ) {
-    clearTimeout(debounceTimeoutId);
-    debounceTimeoutId = window.setTimeout(() => {
-      dispatchUpdateDepartureArrivalTimes(computedDepartureArrivalTimes);
```

Comment:
if we don't use this action anymore, we need to remove `departureArrivalTimes` property everywhere from the app as it isn't ued anywhere.